### PR TITLE
master-m-0817

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-60992

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.testcase [hunk 1]
```diff
@@ -517,10 +966,129 @@ definition {
...
+
+		AssertElementNotPresent(
+			key_attribute = "1",
+			locator1 = "JSFShowcase#MODEL_VALUE_2");
+
+		// SelectOneListboxInstant-ajax
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/guest/jsf-showcase/-/jsf-tag/h/selectonelistbox/instant-ajax");
+
+		Click(
```
Comment (Hanlf) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.testcase]: Able to use JSFShowcase#selectInstantAjax

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.testcase [hunk 2]
```diff
@@ -536,6 +1104,70 @@ definition {
...
+
+		AssertElementNotPresent(
+			key_attribute = "1",
+			locator1 = "JSFShowcase#MODEL_VALUE_2");
+
+		// SelectOneMenuInstant-ajax
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/guest/jsf-showcase/-/jsf-tag/h/selectonemenu/instant-ajax");
+
+		Click(
```
Comment (Hanlf) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.testcase]: can use JSFShowcase#selectInstantAjax

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.macro [hunk 3]
```diff
@@ -193,6 +193,68 @@ definition {
 		AssertElementNotPresent(locator1 = "JSFShowcase#REQUIRED_ERROR_MESSAGE");
 	}
 
+	macro selectError {
```
Comment (Hanlf) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/liferayfaces/JSFShowcase.macro]: Combine this macro into select macro

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
Great job! 
Send pull here: https://github.com/Hanlf/liferay-portal/pull/179

**Pull request closed without merging**